portfolio/src/main/webapp/style.css

```diff
@@ -1,206 +1,202 @@
 @import url('https://fonts.googleapis.com/css2?family=Crimson+Text&display=swap');
 
 body{
-    background-color: rgb(17, 15, 15);
-    color: #fcf4f4;
-    margin-left: auto;
-    margin-right: auto;
-    max-width: 800px;
+  background-color: rgb(17, 15, 15);
+  color: #fcf4f4;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 800px;
 }
 
 .contact-me{
-    color: black;
-    background: #F2F2F2;
-    border-radius: 3px;
-    height: fit-content;
-    margin: 20px auto;
-    opacity: 0.7;
-    width: 500px;
+  color: black;
+  background: #F2F2F2;
+  border-radius: 3px;
+  height: fit-content;
+  margin: 20px auto;
+  opacity: 0.7;
+  width: 500px;
 }
 
 .content {
-    column-gap: 0px;
-    display: grid;
-    grid-template-columns: 550px 200px auto;
-    margin: 30% auto 0px;
+  column-gap: 0px;
+  display: grid;
+  grid-template-columns: 550px 200px auto;
+  margin: 30% auto 0px;
 }
 
 .content-body{
-    align-self: center;
-    left: 20px;
-    padding: 10px 25px;
-    position: relative;
+  align-self: center;
+  left: 20px;
+  padding: 10px 25px;
+  position: relative;
 }
 
 .content-body > h1{
-    font-size: 32px;
+  font-size: 32px;
 }
 
 .content-body .main-content-body{
-    font-size: 22px;
+  font-size: 22px;
 }
 
 .content-body > p{
-    font-size: 16px;
+  font-size: 16px;
```

Comment:
nit: look into rem and em rather than px! (They can resize based off of sizes set in root and parent divs respectfully)

Reply:
> nit: look into rem and em rather than px! (They can resize based off of sizes set in root and parent divs respectfully)

Thanks for the recommendation @Lev-Stambler . I will look into it and update styles respectively

Reply:
+1 to what Lev said. rem is what we prefer internally: num_pixels/16 = rem
Like Lev said that helps with resizing :)